### PR TITLE
EWL-5392 create bio page

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-5392: SG2 | Create Bio Page](https://issues.ama-assn.org/browse/EWL-5392)

## Description
Create bio page along with all the components needed to create it. The components include: 
- [EWL-5410: SG2 | Create Bio Page Stub](https://issues.ama-assn.org/browse/EWL-5410)
- [EWL-5399: SG2 | Content Label molecule](https://issues.ama-assn.org/browse/EWL-5399)
- [EWL-5393: SG2 | Bio Section organism](https://issues.ama-assn.org/browse/EWL-5393)
- [EWL-5391: SG2 | Bio Image with Body](https://issues.ama-assn.org/browse/EWL-5391)

## To Test
- [x]  ` gulp serve`
- [x] visit http://localhost:3000/?p=pages-people-bio
[EWL-5392: SG2 | Create Bio Page](https://issues.ama-assn.org/browse/EWL-5392)
- [x] observe the bio page with a masthead with subtitle
- [x] observe the masthead with subtitle is full width of page [EWL-5289: SG2 | Create Add two column right full width](https://issues.ama-assn.org/browse/EWL-5289)
- [x] observe the bio image with body section in the center of the page with headers italicized and a list of items in a description list. [EWL-5393: SG2 | Bio Section organism](https://issues.ama-assn.org/browse/EWL-5393)
- [x] observe the content as gated line with the lock next to the words "Conflict Of Interest:"
[EWL-5399: SG2 | Content Label molecule](https://issues.ama-assn.org/browse/EWL-5399)
- [x] observe the bio with image organism has an image on left, bio sections and long paragraph in the center and a membership block with a tool block under it
 [EWL-5410: SG2 | Create Bio Page Stub](https://issues.ama-assn.org/browse/EWL-5410)
- [x] observe under the bio with image organism is row of tags and four up teaser under it 
- [ ] - Did you test in IE 11? Yes

## Visual Regressions
Ran backstop test and added new images refs for:

- people bio page
- content as label
- bio section
- body section with image

## Relevant Screenshots/GIFs
screen shot 2018-07-05 at 9 03 54 pm

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Review by labbydev — CHANGES_REQUESTED** [review]
@designerneil it looks like the page isn't collapsing correctly at mobile (see attached screenshot). Also according to the Zeplin files the conflict of interest just shows the lock on smaller screen sizes instead of going to two lines. (see screenshots)

Also it looks like there are a number of backstop tests failing because the paths are off. For example I'm getting a fail on Page title because it's looking at `/patterns/01-atoms-page-title-page-title/01-atoms-page-title-page-title.html` and the pattern really lives at `patterns/01-atoms-page-title/01-atoms-page-title.html`. This might just need to be updated with `develop`?

**Mobile**
screen shot 2018-07-06 at 11 27 49 am


**Conflict of Interest**
screen shot 2018-07-06 at 11 28 40 am


**Conflict of Interest Zeplin**
screen shot 2018-07-06 at 11 28 55 am
screen shot 2018-07-06 at 11 28 49 am

**Comment by labbydev** [comment]
@designerneil the responsive issue has not been addressed.

**Comment by designerneil** [comment]
screen shot 2018-07-06 at 12 28 56 pm

**Comment by designerneil** [comment]
@labbydev are you single a double or single column in mobile view?

File: styleguide/source/assets/scss/04-templates/_two-column-right--full-width.scss [hunk 1]
```diff
@@ -35,4 +35,4 @@
       grid-row: 2 / 2;
     }
   }
-}
+}
```
Comment (labbydev) [inline styleguide/source/assets/scss/04-templates/_two-column-right--full-width.scss:38]: There is no mobile specific breakpoint happening here. And therefore [this line](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/322/files#diff-e3cc4b1c13088567a514c053188bba96R17) is active on mobile. Meaning that I'm seeing `3fr 1fr`	when I should be seeing full width.

**Pull request closed without merging**